src/LocalCoverage.jl

```diff
@@ -168,13 +168,12 @@ end
 
 format_gap(gap) = length(gap) == 1 ? "$(first(gap))" : "$(first(gap)) - $(last(gap))"
 function format_line(summary)
-    hcat(
-        summary isa PackageCoverage ? "TOTAL" : summary.filename,
-        @sprintf("%3d / %3d", summary.lines_hit, summary.lines_tracked),
-        isnan(summary.coverage) ? "-" : @sprintf("%3.0f%%", summary.coverage),
-        summary isa PackageCoverage ? "" :
-        join(map(format_gap, summary.coverage_gaps), ", "),
-    )
+    name = (summary isa PackageCoverage ? "TOTAL" : relpath(summary.filename))
+    lines_hit = @sprintf("%3d / %3d", summary.lines_hit, summary.lines_tracked)
+    coverage = (isnan(summary.coverage) ? "-" : @sprintf("%3.0f%%", summary.coverage))
+    gaps = summary isa PackageCoverage ? "" :
+        join(map(format_gap, summary.coverage_gaps), ", ")
+    return hcat(name, lines_hit, coverage, gaps)
```

Comment:
Given that all arguments are strings, would something like
```suggestion
    return [name, lines_hit, coverage, gaps]
```
be sufficient?

Reply:
It would have to be `[name lines_hit coverage gaps]`, that is, a 1×4 matrix. I just tested it with the vector version (with commas), and that would give a `LoadError: The number of columns in the header must be equal to that of the table.`

I don't know, I'm always feeling a bit weird about Julia's matrix syntax, so I'd have a slightly preference for `hcat`, but I'm ok with whatever you prefer.

Reply:
You can just use NamedTuples as rows, see #39.